x/distribution/spec/04_messages.md

```diff
@@ -74,19 +52,16 @@ func WithdrawDelegationReward(delegatorAddr, validatorAddr, withdrawAddr sdk.Acc
 ```
 
 
-## MsgWithdrawValidatorRewardsAll
+### Withdraw Validator Rewards All
 
 When a validator wishes to withdraw their rewards it must send
-`MsgWithdrawValidatorRewardsAll`. Note that parts of this transaction logic are also
+array of `MsgWithdrawDelegatorReward`. Note that parts of this transaction logic are also
```

Comment:
```suggestion
an array of `MsgWithdrawDelegatorReward`. Note that parts of this transaction logic are also
```

Reply:
Plus, in which context it must send this array since there's no more actual `MsgWithdrawValidatorRewardsAll`?